qtpy/uic.py

```diff
@@ -81,11 +81,17 @@
     if PYSIDE:
         from PySide.QtCore import QMetaObject
         from PySide.QtUiTools import QUiLoader
-        from pysideuic import compileUi
+        try:
+            from pysideuic import compileUi
+        except ModuleNotFoundError:
```

Comment:
`ModuleNotFoundError` is not available in Python 2. Please change it for `ImportError` instead.